src/cart/crt_context.c

```diff
@@ -680,7 +680,8 @@ crt_req_timeout_reset(struct crt_rpc_priv *rpc_priv)
 	rc = crt_req_timeout_track(rpc_priv);
 	D_MUTEX_UNLOCK(&crt_ctx->cc_mutex);
 	if (rc != 0) {
-		D_ERROR("crt_req_timeout_track(opc: %#x) failed, rc: %d.\n",
+		RPC_ERROR(rpc_priv,
+			"crt_req_timeout_track(opc: %#x) failed, rc: %d.\n",
```

Comment:
Just curious...could we make the rc a required argument to RPC_ERROR?

Reply:
we could, but in few places RPC_ERROR is called where there is no explicit rc. not sure it would buy us much either to force rc everywhere